client/src/Movies/MovieList.js

```diff
@@ -11,20 +12,22 @@ export default class MovieList extends Component {
 
   componentDidMount() {
     axios
-      .get('http://localhost:5000/api/movies')
+      .get("http://localhost:5000/api/movies")
       .then(response => {
         this.setState(() => ({ movies: response.data }));
       })
       .catch(error => {
-        console.error('Server Error', error);
+        console.error("Server Error", error);
       });
   }
 
   render() {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <Link to={`/movies/${movie.id}`} className="movieDetailsLink">
```

Comment:
```suggestion
          <Link to={`/movies/${movie.id}`} key={movie.id} className="movieDetailsLink">
```
The key needs to move to this element now, it should be on the outermost element being created by the map